src/components/Cluster/ClusterDetail/Ingress/InstallIngressButton.tsx

```diff
@@ -116,6 +121,12 @@ const InstallIngressButton: React.FC<IInstallIngressButtonProps> = ({
         <Text>
           {installedIngressApp ? (
             '🎉 Ingress controller installed. Please continue to the next step.'
+          ) : clusterIsNotReady ? (
```

Comment:
Linking multiple ternary operators like this really hurts the readibility for me. Do you mind refactoring this into multiple checks?